_tests/unit_tests/modules/footer-button-spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'The Footer Return-to-Top Button', function() {
+
+  var $;
+
+  jsdom();
+
+  before( function() {
+    $ = require( 'jquery' );
+
+  } );
+
+  beforeEach( function() {
+    // Adding a simplified version of the thing we want to test.
+    // Then calling the jQuery to test after
+    $( 'body' ).html( $( '<a class="js-return-to-top"' +
+                            'href="http://www.google.com/">' +
+                            'Back to top' +
+                          '</a>' ) );
+    require( '../../../src/static/js/modules/footer-button.js' ).init();
```

Comment:
I'm pretty sure an absolute path could be used here to shorten the path slightl: `require( '/src/static/js/modules/footer-button.js' ).init();`. Off the top of my head I don't see an issue with that.